blocks/footer/__author-logo/footer__author-logo.css

```diff
@@ -0,0 +1,15 @@
+.footer__author-logo{
+  width: 27px;
+  height: 34px;
+  color: rgba(210, 40, 86, 1);
+}
+
+.footer__author-logo:hover{
+  color: rgba(40, 54, 69, 1);
+  transition: color 0.5s ease-in;
```

Comment:
@ElenaPodv once you've implemented the previous adjustments, please revisit the `footer__author-logo`, `footer__link`, and `logo2` elements to confirm that the `transition` property is applied directly to the elements and not to their states and set it to `all .7s ease` everywhere. 

```CSS
.element {
    /* Other styles for the element */
    transition: all .7s ease; /* Apply transition directly to the element */
}

.element:hover {
    /* Include only styles that actually change here, such as border, opacity, or color */
    /* Avoid placing the transition property here */
}

.element:active {
    /* Include only styles that actually change here, such as border, opacity, or color */
    /* Avoid placing the transition property here */
}
```

Reply:
Done